lang/en.lyaml

```diff
@@ -1332,6 +1334,10 @@ en:
         debug:
           describe: "Set log level to debug"
     configMigrate:
+      migrationHeader: "Migrate your existing configuration file to the new global location?"
+      migrationDescription: "This action will create a new global configuration file at the root of your working directory and rename the deprecated config to {{ archivedConfigPath}}. If any conflicting values are detected, we will prompt you to reconcile them unless the \"--force\" flag is supplied."
+      mergeHeader: "Merge your existing configuration file with the global config?"
+      mergeDescription: "This action will merge your deprecated config file with the global config file located at the root of your working directory. It will also rename the deprecated config to {{ archivedConfigPath }}. If any conflicting values are detected, we will prompt you to reconcile them unless the \"--force\" flag is supplied."
       migrateConfigPrompt: "Migrate config?\n From deprecated config file: {{ deprecatedConfigPath }}\n To new global config file: {{ globalConfigPath }}\n (This action will delete the deprecated config file).\n"
```

Comment:
Ian suggested that we unbold the "From" and "To" here, but unfortunately, the styling (ie. the bold text) is controlled directly by `inquirer`.